File: noi.openjudge.cn/ch0103/10.cpp
// http://noi.openjudge.cn/ch0103/solution/38830979/

#include<iostream>
using namespace std;
int main() {
    float r1,r2,R;
    cin >> r1 >> r2;
    R = 1 / (1 / r1 + 1 / r2);
    printf("%0.2lf\n",R);
}

File: noi.openjudge.cn/ch0107/10.py
# http://noi.openjudge.cn/ch0107/solution/39973153/

s1 = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
s2 = 'VWXYZABCDEFGHIJKLMNOPQRSTU'
d = dict(zip(s1, s2))
s = input()
for c in s:
    if c not in d:
        print(c, end='')
    else:
        print(d[c], end='')
print()
